packages/api/db/migration/20240504185218_create_soil_amendement_task_products_table.js

```diff
@@ -101,6 +101,7 @@ export const up = async function (knex) {
     table.enu('molecular_compounds_unit', molecularCompoundsUnits);
     // Dry matter content is currently 100 - moisture content
     table.decimal('moisture_content_percent', 36, 12);
+    table.check('moisture_content_percent <= 100', [], 'moisture_percent_check');
```

Comment:
@SayakaOno @kathyavini  -- Looks like I added this to the wrong PR 😢 , any preference for how I handle it? 

- new migration
- rollback beta and edit the main migration?

Reply:
No preference. Rollback + amend is nice to keep all the table setup in the same file. But a new migration to add a check is also very readable.

Reply:
I thought I checked that all my comments had been addressed...!
I think new migration is good too!

Reply:
Probably my bad, sometimes I resolve comments if I am 99% sure I did what was asked haha